### DISCUSSION TITLE
Lighthouse Relative Height

### DISCUSSION THREAD

**Discussion by Itai51** [bitcraze/discussions#1145] [Q&A] [answered]
Hello bitcraze community,

I am hanging up four lighthouses in a room with some walls which are clear, and some walls with rafters or cabinets at different heights. All of these points are >40 cm above my desired flight area, so no problem there. However, the rafters along one wall and the cabinets along another are at different heights (~30 cm difference). 

My question is: is it OK to place my different lighthouses at different heights, as long as they are all above the flight area?

Thank you!

**Comment by knmcguire** [comment, marked as answer]
Hi! Yes that is absolutely no problem. We have the same setup in our own meetingroom where there is about 50 cm difference in height between the two basestations. The geometry estimation is able to handle this

**Reply by Itai51** [reply]
Thank you!
